OpenRA.Mods.Common/UpdateRules/Rules/20191117/ReplaceAttackTurnDelay.cs

```diff
@@ -0,0 +1,67 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2019 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using OpenRA.Mods.Common.Traits;
+
+namespace OpenRA.Mods.Common.UpdateRules.Rules
+{
+	public class ReplaceAttackTurnDelay : UpdateRule
+	{
+		public override string Name { get { return "Removed AttackTurnDelay from AttackAircraft and replaced with AttackRunLength."; } }
+		public override string Description
+		{
+			get
+			{
+				return "Removed AttackTurnDelay which defines the strafing run in ticks from AttackAircraft and replaced it with AttackRunLength"
```

Comment:
Minor nit: Can we please change this to:
```csharp
return "Removed AttackTurnDelay which defines the strafing run in ticks from AttackAircraft and\n"
	+ "replaced it with AttackRunLength which defines the strafing run in WDist distance.";
```
The line is too long to print as a single line in the updater output.

Reply:
Fixed.